package.json

```diff
@@ -135,6 +139,7 @@
     "stream-browserify": "^3.0.0",
     "tailwindcss": "^3.0.15",
     "terser-webpack-plugin": "5.3.0",
+    "ts-node": "^10.4.0",
```

Comment:
To support `jest.config.ts` and `jest.setup.ts`. Got the tip via jest terminal-output. There might be a better way to support this, not sure.